cric-predict/src/Components/CreateGroup.js

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import '../Stylesheets/App.css';
+import axios from 'axios';
+import {Button, Modal, Form, Row, Col}  from 'react-bootstrap';
+
+
+export default class CreateGroupsComponent extends React.Component  {
+
+        constructor(){
+            super();
```

Comment:
can't we omit `super()` call here?

Reply:
I can't use 'this' if I omit 'super()' so I have to mention it